zombies.js

```diff
@@ -178,7 +257,20 @@
  * @name eat
  * @param {Food} itemToEat  The food item to eat.
  */
-
+eat(itemToEat) {
+  let chosenFood = this.getPack().indexOf(itemToEat); 
+  if(itemToEat instanceof Food && chosenFood !== -1) {  
+    if((itemToEat.energy + this.health) > this.getMaxHealth()) {
```

Comment:
formatting.

Also, as a small optimization, you do not need to call `this.discardItem(itemToEat)` twice, once in the if and once in the else if.